src/_includes/banner.html

```diff
@@ -6,5 +6,9 @@
       Flutter 1.0 has been released!
       <a href="{{fl_url}}" target="_blank" rel="noopener">Learn more</a>.
     </p>
+    <p>
+      Please take our 
+      <a href="https://google.qualtrics.com/jfe/form/SV_bfowrKmM2DjD5tj?Source=Website">survey</a> to help improve Flutter!
+    </p>
```

Comment:
How about:
> Help improve Flutter by taking our [survey](#)!

@sfshaza2 @kwalrath, WDYT?

Reply:
Or
> Help improve Flutter! Take our [survey](#)

Reply:
I like your shorter suggestions, Patrice. I have no preference for A or B, though.  :)